src/components/Result/ResultContent/OriginalContent.tsx

```diff
@@ -17,23 +16,10 @@ const OriginalContent = ({
   };
   return (
     <Container>
-      {contentType === "article" && (
-        <div className="mx-auto mb-12 max-w-[75rem] animate-fadeIn rounded-[20px] border-2 border-primary bg-white py-12 px-8 text-lg font-semibold md:my-20 md:p-20">
-          <a href={getStringOrFirst(content)} target="_blank" className="hover:underline" rel="noreferrer">
-            {getStringOrFirst(content)}
-          </a>
-        </div>
-      )}
-      {contentType === "song" && (
-        <div className="mx-auto mb-12 max-w-[75rem] rounded-[20px] border-2 border-primary bg-white py-12 px-8 text-xl font-semibold md:my-20 md:p-20">
-          {capitalize(songDetails || getStringOrFirst(content))}
-        </div>
-      )}
-      {contentType === "text" && (
-        <div className="mx-auto mb-12 max-w-[75rem] rounded-[20px] border-2 border-primary bg-white py-12 px-8 md:my-20 md:p-20">
-          {getStringOrFirst(content)}
-        </div>
-      )}
+      <article className=" mx-auto mb-12 max-w-[75rem] rounded-[20px] border-2 border-primary bg-white py-12 px-8 md:my-20 md:p-20">
+        {contentType === "song" && <div>{capitalize(songDetails ?? "")}</div>}
```

Comment:
We could probably add some styling to this to separate it out from the song lyrics?

Reply:
you mean like the summary vs song meaning title in the summary page? Or something else?

Reply:
oh I just meant if we could make the font size for the `Song Title -  Artist Name` a bit larger than the lyrics : 
<img width="1458" alt="image" src="https://user-images.githubusercontent.com/52301822/221981500-1697fdc8-1db8-4e45-a5ee-7dfd14376a0a.png">